jena-arq/src/test/java/org/apache/jena/riot/lang/TestTurtleTerms.java

```diff
@@ -237,6 +241,41 @@
     @Test public void turtle_152() { parse("[ a <y> ; a <z> ] . ") ; }
     @Test public void turtle_153() { parse("[ a <z>, <z1> ] . ") ; }
 
+    // fatal characters
```

Comment:
These are Turtle tests and with the layers over the tokenizer, the tests can't tell where RiotException is coming from. e.g. `turtle_155` - the URI is legal, it sends at `...with_>` leaving junk and error "Triples not terminated by DOT" from the Turtle parser.

The tokenizer tests are in `TestTokenizer`. There is a function `testExpectWarning` that handles testing for warnings.

If you want to leave Turtle behavior tests in, then fine - but the tokenizer is used in various places so it needs its own tests for behaviour.

Reply:
> The tokenizer tests are in `TestTokenizer` etc.

Thanks for the feedback. I started with Turtle parsing tests because that was how the problem manifested itself to me. I have removed these tests and put the single relevant test in `TestTokenizer`, where it belongs.